tests/test_filter_jobs.py

```diff
@@ -11,10 +11,16 @@
 def test_filter_jobs(base_url, selenium):
     """Open resultset page and filter for platform"""
     page = TreeherderPage(selenium, base_url).open()
-    platform = u'Linux'
+    all_builds = page.all_builds
 
+    platform = u'Linux'
     page.filter_by(platform)
-    assert platform in page.result_sets[0].builds[0].platform_name
+    filtered_builds = page.all_builds
+
+    buildname = page.result_sets[0].builds[0].platform_name
+    assert platform in buildname
+
+    assert all_builds > filtered_builds
```

Comment:
Hmm.. this worked for me. Why would filtered builds equal all builds? That suggests a bug considering we're testing that the builds are being filtered. Or are you saying that the filter might not impact the builds that happen to be displayed on staging? If so, I really think we need to stop writing tests that are so heavily dependent on the data that happens to be available - they're either going to be flaky or too forgiving to provide value.